### PR TITLE
MINOR FIX : Teacher Attachment Service Exception Add

### PR DESCRIPTION
**Pull request by tharanisantosh**

### CODE REVIEW

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/TeacherAttachments/TeacherAttachmentServiceTests.Exceptions.Remove.cs [hunk 1]
```diff
@@ -154,7 +154,11 @@ public async Task ShouldThrowServiceExceptionOnRemoveWhenExceptionOccursAndLogIt
             Guid someAttachmentId = randomAttachmentId;
             Guid someTeacherId = randomTeacherId;
             var serviceException = new Exception();
-            var expectedTeacherAttachmentException = new TeacherAttachmentServiceException(serviceException);
+
+            var failedTeacherAttachmentServiceException =
+                new FailedTeacherAttachmentServiceException(serviceException);
+
+            var expectedTeacherAttachmentException = new TeacherAttachmentServiceException(failedTeacherAttachmentServiceException);
```
Comment (ShriHumrudha) [inline OtripleS.Web.Api.Tests.Unit/Services/Foundations/TeacherAttachments/TeacherAttachmentServiceTests.Exceptions.Remove.cs:161]: Break this into two lines

File: OtripleS.Web.Api/Models/TeacherAttachments/Exceptions/FailedTeacherAttachmentServiceException.cs [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+using System;
+using Xeptions;
+
+namespace OtripleS.Web.Api.Models.TeacherAttachments.Exceptions
+{
+    public class FailedTeacherAttachmentServiceException : Xeption
+    {
+        public FailedTeacherAttachmentServiceException (Exception innerException)
+            :base(message:"Failed teacher attachment service error occured.", innerException)
```
Comment (ShriHumrudha) [inline OtripleS.Web.Api/Models/TeacherAttachments/Exceptions/FailedTeacherAttachmentServiceException.cs:14]: Run code cleanup.
: base(message: "Failed....

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/TeacherAttachments/TeacherAttachmentServiceTests.Exceptions.Remove.cs (hunk 1 above)
Comment (tharanisantosh) [reply OtripleS.Web.Api.Tests.Unit/Services/Foundations/TeacherAttachments/TeacherAttachmentServiceTests.Exceptions.Remove.cs]: Comment Resolved.

File: OtripleS.Web.Api/Models/TeacherAttachments/Exceptions/FailedTeacherAttachmentServiceException.cs (hunk 2 above)
Comment (tharanisantosh) [reply OtripleS.Web.Api/Models/TeacherAttachments/Exceptions/FailedTeacherAttachmentServiceException.cs]: Comment Resolved.

**Review by ShriHumrudha — APPROVED** [review]

**Pull request merged**